pkg/documentation/generator/description.go

```diff
@@ -73,8 +74,8 @@ func createDescriptionSection(stepData *config.StepData) string {
 		description += headlineGHA
 		description += fmt.Sprintf("%v```\n", spacingTabBox)
 		description += fmt.Sprintf("%vsteps:\n", spacingTabBox)
-		description += fmt.Sprintf("%v  - name: your preferred name for the step\n", spacingTabBox)
-		description += fmt.Sprintf("%v    uses: SAP/project-piper-action@main\n", spacingTabBox)
+		description += fmt.Sprintf("%v  - uses: SAP/project-piper-action@releaseCommitSHA\n", spacingTabBox)
```

Comment:
Can we use `@1` here as well?

Reply:
We can
It seems the recommendation from GH team is to use SHA as it is safer

Reply:
We can't.

Reply:
ah, really?

Reply:
```
Error: Unable to resolve action `SAP/project-piper-action@1`, unable to find version `1`
```

Reply:
Good to know